modules/core/src/main/scala/org/tessellation/infrastructure/snapshot/GlobalSnapshotConsensusFunctions.scala

```diff
@@ -120,7 +120,7 @@ object GlobalSnapshotConsensusFunctions {
             snapshotContext,
             lastActiveTips,
             lastDeprecatedTips,
-            rewards.distribute(lastArtifact, snapshotContext.balances, _, trigger),
+            rewards.distribute(lastArtifact, snapshotContext.balances, _, trigger, Set.empty),
```

Comment:
imo you should pass `events` here, not the `Set.empty`

Reply:
I think @ryle-goehausen asked me to pass empty set...

Reply:
isn't it misleading that I expect it to be a set of events and I got an empty list? seems to be a bug 🤔

Reply:
Right now the only purpose of this event set is for use in an application rewards function -- we're not calculating any rewards from any other events yet -- if in the future we do so we can pass something here.

Reply:
If this events is easily in scope here however, there's no problem passing it so long as it's not actually affecting any code. I was fine with empty set when we were talking about the other object typed differently. Here now after updating the types its fine to pass in (so long as we aren't using it.)

Reply:
Done